server/src/main/java/io/crate/rest/action/HttpError.java

```diff
@@ -149,8 +148,6 @@ public static HttpError fromThrowable(Throwable throwable) {
             httpErrorStatus = HttpErrorStatus.REPOSITORY_WITH_SAME_NAME_EXISTS_ALREADY;
         } else if (throwable instanceof InvalidSnapshotNameException) {
             httpErrorStatus = HttpErrorStatus.SNAPSHOT_WITH_SAME_NAME_EXISTS_ALREADY;
-        } else if (throwable instanceof UnavailableShardsException) {
-            httpErrorStatus = HttpErrorStatus.ONE_OR_MORE_SHARDS_NOT_AVAILABLE;
         } else if (throwable instanceof ElasticsearchException ex) {
             httpErrorStatus = ex.httpErrorStatus();
```

Comment:
It will fall back to ex.httpError Status 
which is `SERVICE_UNAVAILABLE(503)`, for ES `UnavailableShardsException`

and  `ONE_OR_MORE_SHARDS_NOT_AVAILABLE(HttpResponseStatus.INTERNAL_SERVER_ERROR,5002)` had a different error code. Is it okay?

Reply:
I don't follow. It uses the same `httpErrorStatus` as before.

Reply:
Ah right, it's just that ONE_OR_MORE_SHARDS_NOT_AVAILABLE won't be returned anymore but ES exception is same